src/Pagination/Goto/Goto.components.tsx

```diff
@@ -0,0 +1,63 @@
+// VENDOR
+import * as React from 'react';
+import classNames from 'classnames';
+import styled from '@xstyled/styled-components';
+// ANCHOR
+import { Input } from '../..';
+
+// Todo: this component needs work for mobile and for accessibility.
+
+const EventKeyCodes = {
+    ENTER: 13,
+};
+
+interface GotoProps {
```

Comment:
Does this need the HTML attrs from React?

Reply:
Hmm
```
Interface 'GotoProps' incorrectly extends interface 'HTMLAttributes<HTMLDivElement>'.
  Types of property 'onSubmit' are incompatible.
    Type '(arg: number) => any' is not assignable to type '(event: FormEvent<HTMLDivElement>) => void'.
      Types of parameters 'arg' and 'event' are incompatible.
        Type 'FormEvent<HTMLDivElement>' is not assignable to type 'number'.
```